extensions/amp-base-carousel/1.0/test/test-amp-base-carousel.js

```diff
@@ -249,5 +249,38 @@ describes.realWin(
       const buttons = element.shadowRoot.querySelectorAll('button');
       expect(buttons).to.have.length(0);
     });
+
+    it('should go to slide 0 when index is set to 0', async () => {
```

Comment:
It looks like this test was added to 1.0 rather than 0.1, where the fix was made. Please also add a test for the behavior in the `0.1` test suite, and feel free to leave this new test as is since we also want to make sure this behavior is correct in 1.0. :)